### ISSUE TITLE
5E Statblock Importer: Error Foundry VTT

### ISSUE THREAD

**Issue by Shu89** [jbhaywood/5e-statblock-importer#72]
Hello, the follow issues shows up when using the Importer.

5E STATBLOCK IMPORTER: An error has occured. Please report it using the module link so it can get fixed.
fetch	@	foundry.js:68326
notify	@	foundry.js:68223
:gift:call_wrapped	@	libWrapper-wrapper.js:507
notificationNotify	@	utils.js:4032
:gift:call_wrapper	@	libWrapper-wrapper.js:616
:gift:Notifications.prototype.notify#0	@	libWrapper-wrapper.js:189
error	@	foundry.js:68260
(anonymous)	@	sbiWindow.js:70

Using Foundry Version: 11 Stable - 315
DnD 5E - Fifth Edition System 2.4.0

**Comment by jbhaywood** [comment]
Can you give me the text of the statblock you were importing?

**Comment by Shu89** [comment]
Hello,

sure here is the Statblock: made with
https://cros.land/ai-powered-dnd-5e-monster-statblock-generator/

Black Death Zombie
Medium undead, neutral evil
Armor Class 12 (natural armor)
Hit Points 67 (9d10 + 18)
Speed 20 ft.
STR DEX CON INT WIS CHA
15 (+2) 8 (-1) 15 (+2) 3 (-4) 6 (-2) 5 (-3)
Saving Throws None
Skills Perception +0
Damage Resistances None
Damage Immunities Necrotic, Poison
Condition Immunities Poisoned
Senses Darkvision 60 ft., passive Perception 10
Languages Understands Common but can't speak
Challenge 2 (450 XP)
Undead Fortitude. If damage reduces the Black Death Zombie to 0 hit points,
it must make a Constitution saving throw with a DC of 5 + the damage taken,
unless the damage is radiant or from a critical hit. On a success, the
zombie drops to 1 hit point instead.
Disease Carrier. Any creature that starts its turn within 5 feet of the
Black Death Zombie must succeed on a DC 12 Constitution saving throw or
become infected with a debilitating disease. The infected creature has
disadvantage on Constitution saving throws and takes 1d4 necrotic damage at
the start of each of its turns. The disease lasts until the infected
creature receives magical healing or its hit points are restored to full.
Actions
Decaying Slash. The Black Death Zombie makes a melee attack with its
rotting claws, dealing 17 (2d12 + 4) slashing damage.
Plague Bite. The Black Death Zombie lunges forward to bite its target,
causing 13 (2d8 + 4) piercing damage. If the zombie moves at least 10 feet
straight toward the target and hits it with this attack on the same turn,
the target takes an additional 9 (2d8) necrotic damage. The target must
succeed on a DC 14 Constitution saving throw or become infected with the
Black Death's plague.
Contagious Slam. The Black Death Zombie raises its decaying arm and slams
it down on its enemy. The target must succeed on a DC 13 Dexterity saving
throw or take 10 (2d6 + 3) bludgeoning damage and become restrained as the
zombie's diseased flesh sticks to it. The target can repeat the saving
throw at the end of each of its turns, ending the restraint on a success.
Plague Aura. The Black Death Zombie emits a sickly aura of disease within a
20-foot radius. Any creature that starts its turn within this aura must
make a DC 13 Constitution saving throw or take 7 (2d6) poison damage and
become poisoned until the start of its next turn.

Am Di., 21. Nov. 2023 um 04:18 Uhr schrieb James ***@***.***>:

**Comment by Shu89** [comment]
![image](https://github.com/jbhaywood/5e-statblock-importer/assets/50531824/26a67bba-3047-46f8-b908-0afd2a92bb86)
Here is a screen after i used this text

```
Black Death Mass
Large ooze, unaligned
Armor Class 11
Hit Points 59 (7d10 + 21)
Speed 20 ft., climb 20 ft.
STR DEX CON INT WIS CHA
14 (+2) 6 (-2) 16 (+3) 10 (+0) 8 (-1) 2 (-4)
Saving Throws None
Skills Stealth +2
Damage Resistances None
Damage Immunities None
Condition Immunities Blinded, Charmed, Deafened, Frightened, Grappled, Incapacitated, Paralyzed, Petrified, Poisoned, Prone, Restrained, Stunned, Unconscious
Senses Blindsight 60 ft. (blind beyond this radius), passive Perception 9
Languages None
Challenge 2 (450 XP)
Necrotic Aura. Any creature that starts its turn within 5 feet of the Black Death Mass takes 3 (1d6) necrotic damage.
Undead Fortitude. If damage reduces the Black Death Mass to 0 hit points, it must make a Constitution saving throw with a DC of 5 + the damage taken, unless the damage is radiant or from a critical hit. On a success, the Black Death Mass drops to 1 hit point instead.
Spellcasting. The Black Death Mass is a spellcaster. Its spellcasting ability is Intelligence (+0). It can cast the following spells, requiring an action or bonus action to cast:
- Cantrips: Chill Touch, Ray of Sickness
- 1st Level (2 slots): Inflict Wounds, False Life
Actions
Corroding Slap. The Black Death Mass slaps a creature within reach with its gelatinous pseudopod. It hits with a +3 attack bonus and deals 13 (2d8 + 4) bludgeoning damage. The target must succeed on a DC 11 Constitution saving throw or have its AC reduced by 2 until the end of its next turn as its armor corrodes.
Noxious Eruption. The Black Death Mass expels a cloud of acrid gas in a 10-foot radius. Each creature within the area must make a DC 11 Dexterity saving throw, taking 11 (2d6 + 4) poison damage on a failed save, or half as much damage on a successful one. The gas lingers for 1 minute, causing the area to be heavily obscured and any creature that enters or starts its turn within the gas must succeed on a DC 11 Constitution saving throw or be poisoned until the start of its next turn.
Murky Grasp. The Black Death Mass reaches out with its tendrils to grab hold of a creature within range. It makes a melee spell attack against the target, using its +3 attack bonus. On a hit, the target takes 9 (2d6 + 2) necrotic damage and must succeed on a DC 11 Strength saving throw or be grappled by the Black Death Mass. While grappled, the target takes 4 (1d8) necrotic damage at the start of each of its turns.
```

**Comment by Shu89** [comment]
Ups my mistake, sorry.

**Comment by jbhaywood** [comment]
Fixed in v1.81.7.

**Comment by Shu89** [comment]
Awesome, thanks for the update!
Have a wonderful week.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virenfrei.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Am Do., 23. Nov. 2023 um 06:19 Uhr schrieb James ***@***.***>:
